app/config/env.js

```diff
@@ -1,4 +1,24 @@
-'use strict';
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
```

Comment:
probably want the copyright dated from 2019 😄

Reply:
Good catch @epmkelly 👍